### PR TITLE
Sfsf

### PR DESCRIPTION
**Pull request by cdupuis**
fsf

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Name | Required | Default | Description
 sfsf
 [semver]: http://semver.org
   
-#### Running ss
+#### Running ss some changess
```
Comment (cdupuis) [inline README.md]: bla bla

**Comment by cdupuis** [comment]
Pull request auto merge enabled by @cdupuis. [atomist:enable-auto-merge]

**Comment by tanya-coding** [comment]
test test test

**Comment by cdupuis** [comment]
test

**Comment by cdupuis** [comment]
adasdasda

**Comment by cdupuis** [comment]
dadad

**Pull request closed without merging**